plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,751 @@
+  exclude:
+    description:
+      - Exclude specified guest systems (assumes --all)
+    type: str
+  exclude_path:
+    description:
+      - >
+        Exclude certain files/directories (shell globs).
+        Paths starting with '/' are anchored to the container's root,
+        other paths match relative to each subdirectory.
+    choices: ['ignore', 'on']
+    type: str
+  fleecing:
+    description:
+      - Options for backup fleecing (VM only).
+    type: str
+  id:
+    description:
+      - Required if O(state=absent).
+      - If O(state=present), it allow you to set a pattern of id (Example 0(backup-12345678-9123)) if it not set an ID will be generate automaticly.
+      - Rerquired if O(state=present) and you want to update a existing job.
+    type: str
+  ionice:
+    description:
+      - Set IO priority when using the BFQ scheduler.
+      - For snapshot and suspend mode backups of VMs, this only affects the compressor.
+      - A value of 8 means the idle priority is used,
+      - otherwise the best-effort priority is used with the specified value.
+    type: int
+  lockwait:
+    description:
+      - Maximal time to wait for the global lock (minutes).
+    type: int
+  mailnotification:
+    description:
+      - Specify when to send a notification mail
+    choices: ['always', 'failure']
+    type: str
+  mailto:
+    description:
+      - Comma-separated list of email addresses or users that should receive email notifications.
+    type: str
+  maxfiles:
+    description:
+      - Maximal number of backup files per guest system.
+    type: int
+  mode:
+    description:
+      - Backup mode.
+    choices: ['snapshot', 'suspend', 'stop']
+    type: str
+  node:
+    description:
+      - Only run if executed on this node.
+    type: str
+  notes_template:
+    description:
+      - Template string for generating notes for the backup(s).
+      - It can contain variables which will be replaced by their values.
+      - Currently supported are V({{cluster}}), V{({guestname}}), V({{node}}), and V({{vmid}}),but more might be added in the future.
+      - Needs to be a single line, newline and backslash need to be escaped.
+    type: str
+  performance:
+    description:
+      - Other performance-related settings.
+    type: str
+  pigz:
+    description:
+      - Use pigz instead of gzip when V(N>0).
+      - V(N=1) uses half of cores, V(N>1) uses N as thread count.
+    type: int
+  pool:
+    description:
+      - Backup all known guest systems included in the specified pool.
+      - Can not be use with vmid and pool in same job
+    type: str
+  protected:
+    description:
+      - If true, mark backup(s) as protected.
+    type: bool
+  prune_backups:
+    description:
+      - Use these retention options instead of those from the storage configuration.
+    type: str
+  quiet:
+    description:
+      - Be quiet.
+    type: bool
+  remove:
+    description:
+      - Prune older backups according to 'prune-backups'.
```

Comment:
Do you mean the module option?
```suggestion
      - Prune older backups according to O(prune_backups).
```

Reply:
fix in 900e035f651b9a87963eadaf56385fa61fde9433